BunqSdk/Http/RetryHandler.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Polly;
+
+namespace Bunq.Sdk.Http
+{
+    public class RetryHandler : DelegatingHandler
+    {
+        public RetryHandler(HttpClientHandler handler) : base(handler) { }
+
+        protected override Task<HttpResponseMessage> SendAsync(
+            HttpRequestMessage request,
+            CancellationToken cancellationToken) =>
+            Policy
+                .Handle<HttpRequestException>()
```

Comment:
Only do this on the bunq 429 exception please.

Reply:
This is now in the code.

Reply:
Hmm i meant to only retry when this exception is thrown `TooManyRequestsException` Cant you use this exception to catch and handle ?